integration-tests/debezium/pom.xml

```diff
@@ -39,8 +40,11 @@
         <!--     mvn process-resources -Pformat    from the root directory -->
         <mvnd.builder.rule>camel-quarkus-debezium-postgres-deployment,camel-quarkus-debezium-mysql-deployment,camel-quarkus-support-policy-deployment</mvnd.builder.rule>
```

Comment:
`camel-quarkus-debezium-sqlserver-deployment` seems to be missing.

Reply:
@jamesnetherton Both changes are added to the PR.